core/src/main/java/org/apache/accumulo/core/spi/compaction/DefaultCompactionPlanner.java

```diff
@@ -108,6 +109,16 @@
  * number of files that will be included in a single compaction.
  * </ul>
  *
+ * <p>
+ * Starting with Accumulo 2.1.3, this plugin will use the table config option
+ * {@code "table.file.max"}. When a tablet has no compactions running, its number of files exceeds
+ * table.file.max, and system compactions are not finding anything to compact then this plugin will
+ * try to find a lower compaction ratio that will result in a compaction. For example if the
+ * compaction ratio is set to 3, a tablet has 20 files, table.file.max is 15 and no compactions are
+ * planned, then this plugin will find the largest compaction ratio less than 3 that results in a
+ * compaction.
+ * </p>
+ *
```

Comment:
The wording is a bit awkward, suggest changing it to make the criteria clearer.
```suggestion
 * <p>
 * Starting with Accumulo 2.1.3, this plugin will use the table config option
 * {@code "table.file.max"}. 
 * When the following three conditions are met, then this plugin will try to find a lower compaction ratio that will result in a compaction:
 *  1. When a tablet has no compactions running
 *  2. Its number of files exceed table.file.max
 *  3. System compactions are not finding anything to compact.
 *  For example, given a tablet with 20 files, and table.file.max is 15 and no compactions are planned. If the compaction ratio is set to 3, then this plugin will find the largest compaction ratio less than 3 that results in a
 * compaction.
 * </p>
 *
```

Reply:
Thats easier to understand.  I updated the javadoc, made it use HTML for the list and added a 4th condition that the code was checking for.